airbyte-cdk/python/airbyte_cdk/base_python/cdk/streams/core.py

```diff
@@ -79,7 +79,7 @@ def as_airbyte_stream(self) -> AirbyteStream:
 
         if self.supports_incremental:
             stream.source_defined_cursor = self.source_defined_cursor
-            stream.supported_sync_modes.append(SyncMode.incremental)
+            stream.supported_sync_modes.append(SyncMode.incremental) # type: ignore
```

Comment:
this is ignored as it's I'm fairly certain this is assigned. MyPy isn't detecting it for some reason.